Asgardia.Constitution.Proposal.txt

```diff
@@ -116,21 +116,22 @@ Article 8. Key Citizen Rights and Freedoms 
  h. the right to self-governance;
  i. the right to own and inherit property.
 § 6. Children shall not be exploited. 
-§ 7. All citizens have the right to seek legal reprieve in the Court of Justice for any infringement of rights and property. Every person may sue for redress as provided by law from the State or a public entity, in case he has suffered damage through illegal act of any public official. 
+§ 7. All citizens have the right to seek legal reprieve in the Court of Justice for any infringement of rights and property. Every person may sue for redress as provided by law from the State or a public entity, in case he has suffered damage through the illegal acts of any public official. 
 § 8. Asgardian citizens may not be extradited to other states except as defined by the law.
 § 9. Asgardian citizens have the right to create associations and societies based on the law.
-§ 10. Citizen rights and freedoms may only be restricted by the law of Asgardia to the extent required to protect Asgardia’s national sovereignty, ensure Asgardia’s security, enable the performance of its mission and supreme values, and protect the rights of Asgardian citizens.
+§ 10. Citizen rights and freedoms may only be restricted by the Law of Asgardia to the extent required to protect Asgardia’s national sovereignty, ensure Asgardia’s security, enable the performance of its mission and supreme values, and protect the rights of Asgardian citizens.

```

Comment:
I didn't remove the right to peaceful protest. It's already defined in Article 8 §5a: "people’s right to peaceful assembly." The latter statement was redundant.